docs-2.0/2.quick-start/5.start-stop-service.md

```diff
@@ -1,8 +1,8 @@
 # Manage Nebula Graph services
 
-You can use the `nebula.service` script to start, stop, restart, terminate, and check the Nebula Graph services. This topic takes starting, stopping and checking the Nebula Graph services for examples.
+Nebula Graph uses the `nebula.service` script to start, stop, restart, terminate, and check the Nebula Graph services. This topic takes starting, stopping and checking the Nebula Graph services for examples.
```

Comment:
The `nebula.service` script can start, stop, restart, terminate, and check the Nebula Graph services. So the subject of the first sentence cannot be Nebula Graph, which is already the object.